cypress/integration/views/flowview.ts

```diff
@@ -1,38 +1,51 @@
-import defaultPods from "../../../src/data/defaultPods";
-import { defaultHost, defaultPort } from "../../../src/redux/settings/settings.constants";
+import defaultPods from "../../../src/data/defaultPods"
+import {
+  defaultHost,
+  defaultPort,
+} from "../../../src/redux/settings/settings.constants"
 
 describe("The Flow Page", () => {
   beforeEach(() => {
-    cy.visit("/#/flow");
-  });
+    cy.visit("/#/flow")
+  })
 
   it("should have a working settings button", () => {
     cy.dataName("settingsModal").should("not.exist")
     cy.dataName("settingsButton").click()
     cy.dataName("settingsModal").should("exist")
   })
 
+  it("should create a flow and delete it", () => {
+    cy.dataName("newFlowButton").click()
+    cy.dataName("createEmptyFLowButton").click()
+    cy.dataName("deleteFlowButton-1").click()
```

Comment:
Could you verify verify that it's getting deleted in this it block.

Like, verify that a new flow is getting created after clicking on `createEmptyFLowButton`. and then it's getting removed after clicking on `deleteFlowButton-1`

Reply:
Good point!